pkgs/applications/misc/gnss-sdr/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv, fetchFromGitHub
+, armadillo
+, blas
```

Comment:
Looking at their [CMakelists.txt](https://github.com/gnss-sdr/gnss-sdr/blob/master/CMakeLists.txt). It seems they do the following:
- Require CMake finds LAPACK
- Require CMake finds BLAS
- Try to find armadillo
  - If it is not found, build it with the previously found LAPACK and BLAS
  - If it is found, just use it and ignore which LAPACK and BLAS was actually found previously.

So I think that by adding lapack and blas to the derivation, we actually just pass the first two stages and then drop the found lapack and blas. I think we could get this working by just forcing CMake to pass these first two stages via:
```
cmakeFlags = [ "-DBLAS= YES" "-DLAPACK=YES" ];
```